datafusion/substrait/Cargo.toml

```diff
@@ -36,6 +36,8 @@ object_store = { workspace = true }
 prost = "0.12"
 prost-types = "0.12"
 substrait = "0.22.1"
+
+[dev-dependencies]
 tokio = "1.17"
```

Comment:
I move this to dev-dependencies casually. But `datafusion-substrait` and `datafusion-proto` are not supported yet.